.buildkite/scripts/pipelines/pull_request/pipeline.ts

```diff
@@ -60,6 +60,7 @@ const uploadPipeline = (pipelineContent: string | object) => {
 
     if (
       (await doAnyChangesMatch([
+        /^src\/plugins\/controls/,
```

Comment:
@elastic/kibana-presentation team, here I have added this condition so that security solution cypress tests run everytime there is a change in controls plugin. Please let me know if you disagree.

Reply:
That makes perfect sense to me :)